frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/index.tsx

```diff
@@ -44,7 +44,7 @@ const PodCount: React.FC<SpecCapabilityProps> = ({
       })
     }
   >
-    {value} pods
+    {value ?? 0} pods
```

Comment:
The `spec.replicas` default for some workloads like deployments is actually `1` if no value is provided. I'd suggest showing `-` if there is no value instead.